src/main/java/io/fabric8/jenkins/openshiftsync/PipelineJobListener.java

```diff
@@ -161,8 +178,11 @@ private void upsertItemGroup(ItemGroup itemGroup) {
 
 	private void upsertWorkflowJob(WorkflowJob job) {
 		BuildConfigProjectProperty property = buildConfigProjectForJob(job);
-		if (property != null) {
-			logger.info("Upsert WorkflowJob " + job.getName() + " to BuildConfig: " + property.getNamespace() + "/"
+		if (property != null && 
+		    (!BuildConfigWatcher.isDeleteInProgress(
+		            property.getNamespace()+property.getName()))) {
+			logger.info("Upsert WorkflowJob " + job.getName() + 
```

Comment:
indenting?

Reply:
its for format with 80 chars a line thing